src/applications/personalization/profile/components/direct-deposit/FraudVictimAlert.jsx

```diff
@@ -1,14 +1,17 @@
 import React from 'react';
-import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+import AlertBox, {
+  ALERT_TYPE,
+} from '@department-of-veterans-affairs/formation-react/AlertBox';
 import Telephone, {
   CONTACTS,
   PATTERNS,
 } from '@department-of-veterans-affairs/formation-react/Telephone';
 
-const FraudVictimAlert = () => (
+const FraudVictimAlert = ({ status = ALERT_TYPE.CONTIUNE }) => (
```

Comment:
On Direct Deposit v1 this AlertBox has a gray background. But on Direct Deposit v2 it is blue. This new prop adds support for changing the color.

Reply:
CONTIUNE instead of CONTINUE 😃

Reply:
lol... see I tested this to make sure everything still looked fine in Direct Deposit v1 and then I went back and set up this default value with the typo!